contracts/SSVNetwork.sol

```diff
@@ -239,13 +235,15 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
     /********************************/
     function registerValidator(
         bytes calldata publicKey,
-        uint64[] memory operatorIds,
+        uint64[] calldata operatorIds,
```

Comment:
@vadiminc I remember applying calldata _everywhere_ doesnt reduce gas, only with certain functions. Have you tested it represent savings where its applied? thanks